pandas/tests/indexing/test_loc.py

```diff
@@ -600,8 +600,7 @@ def test_loc_setitem_consistency_empty(self):
         expected = DataFrame(columns=["x", "y"])
         expected["x"] = expected["x"].astype(np.int64)
         df = DataFrame(columns=["x", "y"])
-        msg = "will attempt to set the values inplace instead"
-        with tm.assert_produces_warning(FutureWarning, match=msg):
+        with tm.assert_produces_warning(None):
```

Comment:
Even though this operation is a no-op, shouldn't the warnings still apply here? 

cc @jbrockmendel from https://github.com/pandas-dev/pandas/pull/45333

Reply:
Since this is loc, I don’t think that this should was raised a warning

Reply:
Looks like FutureWarnings were applied to `loc` too in https://github.com/pandas-dev/pandas/pull/45333 so I am not sure.

Reply:
Should have answered in more detail:

```
df = DataFrame(columns=["a", "b"], data=[[1.5, 1.5]])
rhs = DataFrame(columns=["a"], data=[[1.5]])
df.loc[:, "a"] = rhs
```

This does not warn, because it is already in place, so setting empty into empty should not warn either. Additionally, there can't really be done anything inplace anyway

Reply:
Okay. Thank makes sense.